rotkehlchen/api/rest.py

```diff
@@ -2049,19 +2088,26 @@ def remove_chain_type_accounts(
 
         return OK_RESULT
 
-    def _get_manually_tracked_balances(self) -> dict[str, Any]:
+    def _get_manually_tracked_balances(self, usd_value_threshold: FVal | None) -> dict[str, Any]:
         db_entries = get_manually_tracked_balances(db=self.rotkehlchen.data.db, balance_type=None)
         balances = process_result(
             {
                 'balances': db_entries,
             },
-
         )
+        # Filter balances if threshold is set
+        if usd_value_threshold is not None:
+            balances['balances'] = [
+                balance for balance in balances['balances']
+                if 'usd_value' in balance and FVal(balance['usd_value']) >
+                  usd_value_threshold
```

Comment:
same comment. The usd_value should be there so no need to check for it. Maybe you can change this whole logic and do it before the call to `balances = process_result(`. The `db_entries` is a list of `ManuallyTrackedBalanceWithValue` and you can check for `ManuallyTrackedBalanceWithValue.value.usd_value`